docs/api-usage.md

```
@@ -0,0 +1,1172 @@
+|------|-----|
+| Base URL | `http://192.129.209.36:3001/v1` |
+| 认证方式 | HTTP Header `Authorization: Bearer <api-key>` |
+| 兼容协议 | OpenAI API (Chat Completions, Models, Video Generations) |
+| 测试 API Key | `sk-qZ9riqHVLChWVgVJXEkgYht3kVvVnnXS9xx9hVjzlcG7nKe9` |
+| 测试 Key 额度 | 500,000,000 单位（约 $500） |
+
+当前入口运行在 2026-05-26 迁移后的新服务器上，由 Coolify 资源 `new-api-video-gateway` 管理。迁移后已用 `grok-video-3` 重新完成真实生成验证，任务 `task_kTOu1dhTCYZvSYtynESiQQz0rEqlHOjO` 已完成，`/content` 下载返回 `200 video/mp4`。
+
+所有请求必须在 HTTP Header 中携带 API Key：
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)